docs/querying/functions.md

```diff
@@ -448,6 +430,7 @@ over time and return an instant vector with per-series aggregation results:
 * `stddev_over_time(range-vector)`: the population standard deviation of the values in the specified interval.
```

Comment:
`the value 1 for any series in the specified interval.`

I think it would be more clear, WDYT?